lib/tpm2_util.c

```diff
@@ -41,6 +41,157 @@
 #include "tpm2_tool.h"
 #include "tpm2_util.h"
 
+#define FILE_PREFIX "file:"
+#define FILE_PREFIX_LEN (sizeof(FILE_PREFIX) - 1)
```

Comment:
I don't think you need this hunk

Reply:
Good catch ... I forgot to strip that out when I ported from master